blog-posts/gatsby-ssr-caching.md

```diff
@@ -0,0 +1,98 @@
+---
+path: "/blog/cache-gatsby-ssr-pages-on-gatsby-cloud/"
+date: "2022-02-15"
+title: "Cache Gatsby SSR pages on Gatsby Cloud"
+featuredImage: images/gatsby-ssr-caching.jpg
+author: "Fabian Dietenberger"
+authorSummary: "Senior Developer at Satellytes"
+teaserText: Learn how to cache Gatsby SSR pages on Gatsby Cloud
+seoMetaText: Learn how to cache Gatsby SSR pages on Gatsby Cloud to boost page speed. Your users won't see a difference to SSG or DSG pages anymore.
+leadboxText: Want to learn more about creating blazing fast pages? Then join us to learn more!
+attribution:
+    creator: Shiro Hatori
+    source: https://unsplash.com/photos/WR-ifjFy4CI
+---
+
+Since Gatsby 4 you can do [server side rendering](https://www.gatsbyjs.com/docs/how-to/rendering-options/using-server-side-rendering/) 
+of pages. This means, the page gets rendered at runtime, and not at buildtime.
+
+When serving a page via SSR, efficient caching becomes crucial as the browser won't show anything until the whole page 
+is rendered on the server and delivered to the browser. Caching can be done on multiple levels in your architecture, in 
```

Comment:
`Caching can be done on multiple` very interesting stuff, but nothing that is common knowledge, imo. 

I think that deserves it's own paragraph. Maybe mention (bullet-points suffice) the different caching layers and explain why caching @ edge is the way to go.

Reply:
Added it